modules/network-observability-quickfilter.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: REFERENCE
+[id="network-observability-quickfilter{context}"]
+= Filtering the network traffic
+By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:
+
+
+.Quick filters
+The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 
+
+.Advanced filters
+You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
+
+[NOTE]
+====
+To understand the rules of specifying the text value, click *Learn More*.
+====
+
+
+.Query Options
+You can use *Query Options* to optimize the search results, as listed below:
+
+** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology view. The default selected value is *Destination*.
+** *Match filters*: You can determine the relation between different filter parameters selected in advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
+** *Limit*: The limit for internal backend queries. Depending upon the matching and filter settings, several queries can be performed under the cover, each with this limit set, resulting in more results after aggregation.
```

Comment:
Not sure what "under the cover" means here

Reply:
It was supposed to mean backend. But I have removed it now as the phrase doesn't make any relevance to the readers.